core/kamon-core/src/main/resources/reference.conf

```diff
@@ -241,6 +241,29 @@ kamon {
       }
     }
 
+    # Change the sampling decision for a trace automatically based on the either the number of errors in a trace, or a
+    # fixed latency threshold. The tail sampler will only work when the `span-reporting-delay` setting is set to a value
+    # greater than zero.
+    #
+    # Important things to keep in mind:
+    #   - The sampling decision override happens only in the current process. This means that any requests sent to other
+    #     services before the sampling decision override happened might be missing from the resulting trace.
+    #   - Try to set `span-reporting-delay` to a value slightly higher than your application's timeout to ensure all the
+    #     local spans related to the trace will still be kept in memory.
+    #
+    tail-sampler {
```

Comment:
I'm wondering if we should call this `local-tail-sampler` or maybe `sampling-decision-overwrite`. I guess tail sampler is going to be easier to find because it is the industry name for this kind of behavior, but I would like to emphasize that this is a local-only feature.

Reply:
+1 to `local-tail-sampler` instead of  `tail-sampler`

Reply:
Got it done.